feature/platform/transceiver/zr_tunable_parameters_test/README.md

```diff
@@ -1,15 +1,25 @@
-# TRANSCEIVER-5: Configuration: 400ZR channel frequency and output TX launch power setting. 
+# TRANSCEIVER-5: Configuration: 400ZR channel frequency, output TX launch power and operational mode setting. 
 
 ## Summary
 
-Validate setting 400ZR tunable parameters channel frequency and output TX
-launch power and verify corresponding telemetry values.
+Validate setting 400ZR tunable parameters channel frequency, output TX
+launch power and operational mode and verify corresponding telemetry values.
 
 ### Goals
 * Verify full C band frequency tunability for 100GHz line system grid.
 * Verify full C band frequency tunability for 75GHz line system grid.
 * Verify adjustable range of transmit output power across -13 to -9 dBm
-  in steps of 1 dB.  
+  in steps of 1 dB.
+* Verify  that the ZR module Host Interface ID and Media Interface ID
+  combination to ZR module AppSel mapping can be configured through the OC
+  `operational-mode`. `operational-mode` is a construct in OpenConfig that masks
+  features related to line port transmission. OC operational modes provides a
+  platform-defined summary of information such as symbol rate, modulation,
+  pulse shaping, etc.
+
+**Note** For standard ZR, OIF 400ZR with C-FEC is the default mode however as we
+move to 400ZR++ and 800ZR, optic AppSel code would need to be configured
```

Comment:
I am not clear on the intent of this note. It seems to suggest the operational-mode is not needed. Additionally, it would suggest a device behavior that when not provided that it should assume C-FEC with OIF 400ZR.

Instead, should we remove this and just expect this to be a requirement to provide the operational-mode and avoid expecting default behaviors?